test/compression.js

```diff
@@ -7,6 +7,16 @@ var http = require('http')
 var request = require('supertest')
 var zlib = require('zlib')
 
+var describeHttp2 = describe.skip
+try {
+  var http2 = require('http2')
+  describeHttp2 = describe
+} catch (err) {
+  if (err) {
+    console.log('http2 tests disabled.')
+  }
```

Comment:
Why would we send a message to the console? Wouldn't it be enough to just ignore it?

Reply:
Made it verbose previously. Yes, we can ignore it, since the test results will mention it as skipped anyway.

Reply:
I’m not going to block this PR just because of a message in the console that will be ignored anyway.